PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -332,6 +332,10 @@ def miniAOD_customizeCommon(process):
     addBoostedTaus(process)
     process.load("RecoTauTag.Configuration.RecoPFTauTag_cff")
     process.load("RecoTauTag.Configuration.HPSPFTaus_cff")
+    #enable correct behaviour of overlap removal in boosted tau seeding
```

Comment:
Please move into RecoTauTag/Configuration/python/boostedHPSPFTaus_cff.py, where ```boostedTauSeed``` is defined

Reply:
Moved as suggested.
I am only a bit afraid that this modifier can be overlooked when preparing UL reMiniAOD one as it is now outside PAT packages...

Reply:
Thank you @mbluj 
The reason to allow modifier-driven changes only in the config which defines the module configuration is exactly because otherwise it could be difficult to track the modifications applied in non related packages.